sentry-spring-jakarta/src/main/java/io/sentry/spring/jakarta/webflux/ReactorUtils.java

```diff
@@ -36,20 +39,20 @@ public static <T> Mono<T> withSentry(final @NotNull Mono<T> mono) {
    */
   @ApiStatus.Experimental
   public static <T> Mono<T> withSentryNewMainHubClone(final @NotNull Mono<T> mono) {
-    final @NotNull IHub hub = Sentry.cloneMainHub();
+    final @NotNull IScopes hub = Sentry.cloneMainHub();
     return withSentryHub(mono, hub);
   }
 
   /**
-   * Writes the given Sentry {@link IHub} to the {@link Context} and uses {@link
+   * Writes the given Sentry {@link IScopes} to the {@link Context} and uses {@link
    * io.micrometer.context.ThreadLocalAccessor} to propagate it.
    *
    * <p>This requires - reactor.core.publisher.Hooks#enableAutomaticContextPropagation() to be
    * enabled - having `io.micrometer:context-propagation:1.0.2+` (provided by Spring Boot 3.0.3+) -
    * having `io.projectreactor:reactor-core:3.5.3+` (provided by Spring Boot 3.0.3+)
    */
   @ApiStatus.Experimental
-  public static <T> Mono<T> withSentryHub(final @NotNull Mono<T> mono, final @NotNull IHub hub) {
+  public static <T> Mono<T> withSentryHub(final @NotNull Mono<T> mono, final @NotNull IScopes hub) {
```

Comment:
Rename to `scopes`

Reply:
Changed in https://github.com/getsentry/sentry-java/pull/3366